pincer/objects/guild/channel.py

```diff
@@ -177,7 +177,7 @@ class Channel(APIObject):
 
     @classmethod
     async def from_id(cls, client: Client, id: int) -> Channel:
-        data = (await client.http.get(f"/guilds/{id}")) or {}
+        data = (await client.http.get(f"/channels/{id}")) or {}

```

Comment:
I might be wrong, but I think that the first slash of the endpoint can be removed. Since it's already being put there by the http client.